File: benchmarks/builtins/math_functions.py
# Mathematical built-ins - Python
# Tests native math module functions

import math

sum_total = 0

for i in range(1, 10001):
    sqrt_val = math.sqrt(i)
    abs_val = abs(sqrt_val - 50)
    floor_val = math.floor(abs_val)
    ceil_val = math.ceil(abs_val)
    sum_total += floor_val + ceil_val

print(sum_total)

File: benchmarks/builtins/math_functions.js
// Mathematical built-ins - JavaScript
// Tests native Math object functions

let sum = 0;

for (let i = 1; i <= 10000; i++) {
    const sqrtVal = Math.sqrt(i);
    const absVal = Math.abs(sqrtVal - 50);
    const floorVal = Math.floor(absVal);
    const ceilVal = Math.ceil(absVal);
    sum += floorVal + ceilVal;
}

console.log(sum);